### PR TITLE
Introducing github actions

### PR DESCRIPTION
**Pull request by fabricio-aguiar**
https://pulp.plan.io/issues/6067
closes #6067

GH Actions:
https://github.com/fabricio-aguiar/plugin_template/runs/438586416?check_suite_focus=true

### CODE REVIEW

**Comment by dralley** [comment]
Looks good, although we won't want to keep Travis running for long afterwards due to the duplication of the scripts.

File: .travis/before_install.sh [hunk 1]
```diff
@@ -1,12 +1,5 @@
 #!/usr/bin/env bash
 
-# WARNING: DO NOT EDIT!
-#
-# This file was generated by plugin_template, and is managed by bootstrap.py. Please use
-# bootstrap.py to update this file.
-#
-# For more info visit https://github.com/pulp/plugin_template
-
```
Comment (mdellweg) [inline .travis/before_install.sh:9]: Why should this header be removed?
The reference to `bootstrap.py` is wrong though.

**Comment by mdellweg** [comment]
The scripts copied from travis to gh-actions look mostly identical to me.
Would it be feasible to make them ci-agnostic and put them in a common location?

File: .travis/before_install.sh (hunk 1 above)
Comment (fabricio-aguiar) [reply .travis/before_install.sh:9]: this is not from the template, it is the CI config for `plugin_template` itself, and it has some differences from the one generated from template.

**Comment by fabricio-aguiar** [comment]
> The scripts copied from travis to gh-actions look mostly identical to me.
> Would it be feasible to make them ci-agnostic and put them in a common location?

This PR started as PoC, and many things changed after I started it, I believe we will remove travis entirely

**Comment by fabricio-aguiar** [comment]
Waiting github actions features

**Pull request closed without merging**